contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -112,34 +115,47 @@ contract RyskStrategy is BaseStrategy {
             memory depositReceipt = _getDepositReceipt();
         IRyskLiquidityPool.WithdrawalReceipt
             memory withdrawalReceipt = _getWithdrawalReceipt();
+        uint256 currentWithdrawalEpoch = ryskLqPool.withdrawalEpoch();
         // since withdrawal price per share is not updated until the end of the epoch,
         // we need to use the price per share from the previous epoch
-        uint256 withdrawalPricePerShare = ryskLqPool
+        uint256 currentWithdrawalPricePerShare = ryskLqPool
             .withdrawalEpochPricePerShare(ryskLqPool.withdrawalEpoch() - 1);
 
         // shares for pending deposit (not yet minted or not yet updated on the deposit receipt as unredeemed)
         uint256 amountInPendingDepositShares = _getAmountInSharesForPendingDeposit(
                 depositReceipt,
-                withdrawalPricePerShare
+                currentWithdrawalPricePerShare
             );
 
         // shares marked for withdrawal that are now owned by the pool itself
```

Comment:
yo, same as in the description, misleading comment, I'm not sure it's necessary to have it here, it's already explained elsewhere

Reply:
not sure if this is still relevant, but I've made some changes